plugins/importtext_procedure.cc

```diff
@@ -92,6 +92,9 @@ ImportTextConfigDescription::ImportTextConfigDescription()
              "If true, the indexes of the columns will be used to name them."
              "This cannot be set to true if headers is defined.",
              false);
+    addAuto("processExcelFormulas", &ImportTextConfig::processExcelFormulas,
+            "Process formulas in `=\"...\"` format from Excel export of "
```

Comment:
This suggests that we do more that we actually do.  It is not all formulas but only the simple formulas `="n"` where `n` is a numeral.